### PR TITLE
add sort direction toggle

### PR DESCRIPTION
**Pull request by bhavik-soni**
Author: Bhavik

## What changes were made?
- add toggle to change sort order between asc and desc
- make column titles clickable as an alternative to the dropdown

## Why are these changes important/necessary?
- There was a deceiving arrow that didn't work. This change improves leaderboard interactivity

### CODE REVIEW

**Comment by bhavik-soni** [comment]
lint fail

**Comment by elimelt** [comment]
use `npm run lint` to see why dry build fails. also will want to update your branch with main

File: src/types.ts
Comment (elimelt) [inline src/types.ts]: nit: I prefer enum

File: src/components/Leaderboard.tsx
Comment (elimelt) [inline src/components/Leaderboard.tsx]: missed here maybe?

File: src/components/Leaderboard.tsx
Comment (elimelt) [inline src/components/Leaderboard.tsx]: missed here maybe?

File: src/components/Leaderboard.tsx
Comment (bhavik-soni) [reply src/components/Leaderboard.tsx]: missed the file

File: src/components/GithubLeaderboard.tsx
Comment (elimelt) [inline src/components/GithubLeaderboard.tsx]: do we have a type that this string adheres to as a key of?

Comment (bhavik-soni) [reply src/components/GithubLeaderboard.tsx]: I don't think so?

Comment (elimelt) [reply src/components/GithubLeaderboard.tsx]: a lot of the `string`s can be replaced, something like this maybe:

https://github.com/swecc-uw/swecc-leaderboard/commit/d85542ca33fa1917a769b7e953babe02085418b2

although it isn't strictly needed. Can cherry-pick/merge this commit if you want, otherwise lgtm

**Review by elimelt — APPROVED** [review]

**Pull request merged**
